### ISSUE TITLE
Suggestions: Character amount, word count and other information in status bar.

### ISSUE THREAD

**Issue by mknepper** [linuxmint/xed#419]
I was using Xed, which is great, but it could use a couple features. There's no character count displayed in the status bar and there should be a basic plugin for word count. _Vim_ and _Neovim_ both have these as well as other text editors. I'm no programmer but I'm assuming it wouldn't require too much overhead to add. 

**Suggestions:**
- Character Count in Status Bar (next to Columns and INS status)
- Word Count Plugin for Status Bar (somewhere between Columns and INS status)

**Comment by JosephMcc** [comment]
I don't know if we need all this in the statusbar. It's also used for displaying other types of messages. You do know that you can go to Tools->Document Statistics anytime?

**Comment by jedomed** [comment]
I'd also appreciate if there was a way to add word count to the status bar. I know about the document statistics but that's an extra window that has to be opened, and more importantly it needs to be manually updated every time I change something or switch to another file.

**Comment by marcodsad** [comment]
I would like to up this feature request. 

yes, one can go to Tools > Doc. Stats, but that is really a flow-breaker. I love writing in xed because it is distraction free, almost completely. Except the fact that when I need to know how many chars I'm writing, I have to interrupt my writing flow. It's a small thing, but I would not overlook it in terms of usability. It would change a lot the experience, especially for those, like me, writing chars-constrained text on a daily basis.

Thanks for considering it!

**Comment by victorbnl** [comment]
I agree, opening the menu anytime is a real pain. At least having the character count in the status bar would be much better

**Comment by ineuw** [comment]
. . . All that code that begs to be written . . . and not a coder in sight . . .  
No feature, that has been already asked for, and will be asked in the future, is impossible. All requested features exist in one app or another in Linux, but not in a single app. I would rather learn how to code a simple plugin.

**Comment by victorbnl** [comment]
I'd love to write some features myself and directly contribute to the project by making a pull request (which would be more interesting than plugins I think), but it's hard to come to grips with such codebases. I don't have enough experience yet I guess

**Comment by ineuw** [comment]
I believe everything additional is a plugin.

**Comment by victorbnl** [comment]
That makes sense too

**Comment by ineuw** [comment]
If you are interested, I think that gedit plugins are adaptable to xed.

**Comment by victorbnl** [comment]
Great, I'll check it out. Thank you!

**Comment by changenowde** [comment]
For me the gedit plugin did not work in xed. So it would be great to have a work count in the bottom status bar. Otherwise it is a lovely text editor! :-)
